editor/svgcanvas.js

```diff
@@ -3043,7 +3044,7 @@ this.svgToString = function (elem, indent) {
 
       const mozAttrs = ['-moz-math-font-style', '_moz-math-font-style'];
       for (i = attrs.length - 1; i >= 0; i--) {
-        attr = attrs.item(i);
+        attr = attrs[i];
```

Comment:
Can you remove the "attr" variable declared above and declare variables inside the loops?

Reply:
Done.